src/stores/index.js

```diff
@@ -80,15 +80,11 @@ const LobsterStore = Reflux.createStore({
     const gitPrefixLen = gitPrefix.length + 2;
     let gitVersionStr = 'master';
     const portRegex = / [sdbc](\d{1,5})\|/;
-    const stateRegex = /:(initsync|primary|secondary\d*|node\d*)]/;
+    const stateRegex = /(:shard\d*|:configsvr)?:(initsync|primary|mongos|secondary\d*|node\d*)]/;
 
     const colorMap = {};
 
-    const colorList = ['seagreen', 'steelblue',
-      'mediumpurple', 'crimson', 'darkkhaki',
-      'darkgreen', 'rosybrown', 'chocolate',
-      'orangered', 'darkseagreen', 'royalblue',
-      'slategray'];
+    const colorList = ['#5aae61', '#c2a5cf', '#bf812d', '#dfc27d', '#2166ac', '#8c510a', '#1b7837', '#74add1', '#80cdc1'];
```

Comment:
Newlines please. Also, is it possible to add more colors than this? This actually has fewer lines